pinot-transport/src/test/java/com/linkedin/pinot/transport/common/routing/RoutingTableTest.java

```diff
@@ -400,4 +409,95 @@ public void testCombinedKafkaRouting() throws Exception {
     }
     return configs;
   }
+
+  class FakePropertyStore extends ZkHelixPropertyStore<ZNRecord> {
```

Comment:
If this is generic enough, I suggest moving it to a separate file so that it can be reused in other tests. If not, leave this here.